src/error.rs

```diff
@@ -0,0 +1,73 @@
+//! authd(2)
+//!
+//! 1. Launch a DC/OS strict cluster.
+//! 2. Create key pair:
+//! ```
+//! dcos security org service-accounts keypair usi.private.pem usi.pub.pem
+//! ```
+//! 3. Create user strict-usi:
+//! ```
+//! dcos security org service-accounts create -p usi.pub.pem -d "For testing USI on strict" strict-usi}
+//! ```
+//! 4. Download SSL certs:
+//! ```
+//! wget --no-check-certificate -O dcos-ca.crt "$(dcos config show core.dcos_url)/ca/dcos-ca.crt"
+//! ```
+//! 5. Convert the private key
+//! ```
+//! openssl rsa -in usi.private.pem -outform DER -out usi.private.der
+//! ```
+//! 6. Run with `cargo run`
+
+use jsonwebtoken;
+use reqwest;
+use std::{convert, error, fmt, io};
```

Comment:
One more thing, if you want to restrict this more (for nothing else other than to communicate to others at the top of the file you're using the From trait and not Into/TryInto) you can:

`use std::{convert::From, error, fmt, io}`

The rest I'd leave the same since there's so many uses of Error here, and namespacing Result/Formatter with `fmt` has a lot more clarity.